README.md

```diff
@@ -8,7 +8,7 @@ WiFi / BTLE coexistence is implemented but currently only works (to some extend)
 
 On ESP32-S3 only WiFi is currently supported.
 
-THIS CURRENTLY DOESN'T WORK WITH THE XTENSA ENABLED RUST COMPILER 1.63.0.2. Use 1.64.0.0!
+THIS CURRENTLY DOESN'T WORK WITH THE XTENSA ENABLED RUST COMPILER 1.63.0.2. Use 1.65 or better
```

Comment:
Shall we just rewrite this as "minimum support Rust compiler version: 1.65.0.0"?

Reply:
Sounds good! Will do